module/segment/tests/Infrastructure/Handler/Command/CalculateProductCommandHandlerTest.php

```diff
@@ -15,18 +15,15 @@
 use Ergonode\Segment\Infrastructure\Handler\Command\CalculateProductCommandHandler;
 use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
+use Ergonode\Segment\Infrastructure\Service\SegmentProductService;
 
 class CalculateProductCommandHandlerTest extends TestCase
 {
-    /**
-     * @var CalculateProductCommand|MockObject
-     */
-    private $command;
+    private CalculateProductCommand $command;
 
-    /**
-     * @var SegmentQueryInterface|MockObject
-     */
-    private $query;
+    private SegmentQueryInterface $query;
+
+    private SegmentProductService $service;
```

Comment:
I like the fact that property is marked as MockObject and actual class because it gives you the right amount of info. If that's ok with you I'd leave how it is - in this version we don't have info at all that it is a mock.